apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,22 @@
+import { Avatar } from "../components.js";
+import styles from "./styles.module.css";
+
+const Header = (): JSX.Element => {
+	return (
+		<header className={styles["header"]}>
+			<div className={styles["logo-container"]}>
+				<img
+					alt=""
+					className={styles["logo-img"]}
+					src="/assets/images/logo.svg"
+				/>
+				<span className={styles["logo-text"]}>Logo</span>
+			</div>
+			<div>
+				<Avatar />
+			</div>
+		</header>
+	);
+};
+
+export { Header };
```

Comment:
maybe import the image source and then use it in your component

`import logoSrc from "/assets/images/logo.svg";`
...
`src={logoSrc}`